spec/features/user_adds_new_game_spec.rb

```diff
@@ -16,8 +16,15 @@
     expect(page).to have_content("wowgame")
   end
 
-  scenario "game details can't be blank" do
+  scenario 'visitors cannot add a new game' do
+    visit "/games/new"
+    expect(page).to have_content(
+      'You need to sign in or sign up before continuing.'
+    )
+  end
 
+  scenario "game details can't be blank" do
+    sign_up
     visit "/games/new"
     click_button "add game"
     expect(page).to have_content("Playing time can't be blank")
```

Comment:
You probably would want to check that other content like "Minimum Players Can't Be Blank" etc exists too.